index.html

```diff
@@ -158,20 +179,21 @@ <h2 class="header intro-header">About <br>Myself</h2>
                 <li>
                     Skills
                     <span>&#62;</span>
-                </li>                                                               
+                 </li>                                                               
             </ul>
+        </div>
 
-            <div class="form">
-                <h2 class="header">Contact me</h2>
-                <p class="p">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-                <form action="https://formspree.io/f/xdobnydq" method="post">
-                    <input type="text" name="name" placeholder="Name" maxlength="30" required>
-                    <input type="email" name="email" placeholder="E-mail" required>
-                    <textarea name="message" maxlength="500" placeholder="Write your message here" required></textarea>
-                    <button type="submit">Get It Touch</button>
-                </form>
-                <div id="short-line"></div>
-            </div>            
+        <div class="form">
```

Comment:
- [ ] I think it could be a good idea to apply the suggestion about the background image just like in the headline section :pushpin: :raised_hands: